core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineOptions.java

```diff
@@ -87,6 +87,10 @@ private OnlineOptions(Data data) {
         this.sslOptions = data.sslOptions;
         this.wrappedModelControllerClient = data.wrappedModelControllerClient;
         this.isWrappedClient = data.wrappedModelControllerClient != null;
+
+        if (protocol == ManagementProtocol.HTTPS && sslOptions == null) {
```

Comment:
Right, this should have been here already. It should also check for `HTTPS_REMOTING`.